docs/proposal.md

```diff
@@ -4,15 +4,9 @@
 
 Our role: Data Analysts at a consulting firm
 
-Target Audience: Movie enthusiasts & Directors at Movie Production Companies
+Target Audience: Movie enthusiasts who enjoy researching movies
```

Comment:
not sure if keeping this criterion less specific would be better? Hope to get a second opinion!
```suggestion
Target Audience: Movie enthusiasts 
```